src/Controller/Component/FlashComponent.php

```diff
@@ -91,6 +92,10 @@ public function set($message, array $options = [])
             $message = $message->getMessage();
         }
 
+        if (isset($options['escape'])) {
+            $options['params'] += ['escape' => $options['escape']];
```

Comment:
The idea was to be consistent with a few lines on top where it was also merged in this way to avoid overwriting it if it was already set inside the method options (which are of higher priority).